test/datasets/test_fake_dataset.py

```diff
@@ -18,13 +20,19 @@ def test_fake_dataset(with_nan):
             torch_frame.numerical,
             torch_frame.categorical,
             torch_frame.text_embedded,
+            torch_frame.text_tokenized,
         ],
         text_embedder_cfg=TextEmbedderConfig(
             text_embedder=HashTextEmbedder(out_channels), batch_size=None),
+        text_tokenizer_cfg=TextTokenizerConfig(
+            text_tokenizer=WhiteSpaceHashTokenizer(num_hash_bins=12),
+            batch_size=5),
     )
     assert str(dataset) == 'FakeDataset()'
     assert len(dataset) == num_rows
-    assert dataset.feat_cols == ['a', 'b', 'c', 'x', 'y', 'text_1', 'text_2']
+    assert dataset.feat_cols == [
+        'a', 'b', 'c', 'x', 'y', 'text_1', 'text_2', 'text_3', 'text_4'
```

Comment:
at a first glance, I got a bit confused by text_1/2/3/4. would be good to rename these.